packages/lab/src/combo-box-next/ComboBoxNext.css

```diff
@@ -10,4 +10,5 @@
   border-color: var(--salt-selectable-borderColor-selected);
   box-shadow: var(--salt-overlayable-shadow-popout);
   max-height: calc((var(--salt-size-base) + var(--salt-spacing-100)) * var(--comboBoxNext-itemCount, 5));
+  z-index: var(--salt-zIndex-flyover);
```

Comment:
Can we just double check this is the correct z-index token that should be used? It seems quite high

Reply:
```suggestion
  z-index: calc(var(--salt-zIndex-appHeader) - 1);
```

Reply:
@JoanaMMoreira updated to align z-index token used with Dialog/Drawer component